pkg/sloop/store/typed/tables.go

```diff
@@ -28,6 +29,10 @@ type MinMaxPartitionsGetter interface {
 	GetMinMaxPartitions(badgerwrap.Txn) (bool, string, string)
 }
 
+type MinPartitionsGetter interface {
```

Comment:
Should we rename the interface above to be PartitionGetter and add the relevant methods there?

Reply:
changed to PartitionGetter